### PR TITLE
Migrating Old Framework Documentation

### PR DESCRIPTION
**Pull request by h3ndrk**
## Introduced Changes

What it says, still needs some cleanup but at least everything that I had on my stash is now in here.

## ToDo / Known Issues

None

## Ideas for Next Iterations (Not This PR)

None

## How to Test

Look at the docs :wink:

### CODE REVIEW

File: docs/framework/process_entrypoint.md [hunk 1]
```diff
@@ -15,14 +25,5 @@ This concept allows to shutdown gracefully in any case of error or termination r
...
 See [Hardware Interface](./hardware_interface.md) for more information about what the hardware interface initializes.
-At the end, the runtime is started.
-The `main()` function then waits for termination of the runtime which then concludes the process execution.
-
-## Behavior Simulator
-
-The behavior simulator is a special build target which only initializes and starts a subset of the robotic control software.
-It is intended to be executed on the development machine.
-Cancellation and hardware interfaces are not needed and are therefore omitted from initialization in `main()`.
-Instead, the behavior simulator parses command line arguments and dispatches the behavior simulation.
```
Comment (schluis) [inline docs/framework/process_entrypoint.md:28]: Why did you remove this?

**Review by schluis — CHANGES_REQUESTED** [review]
👌🏻

File: docs/setup/git_setup.md [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
 
-The first thing you should do when you install Git is to set your user name and email address. This is important because every Git commit uses this information, and it’s immutably baked into the commits you start creating:
+The first thing you should do when you install Git is to set your user name and email address.
+This is important because every Git commit uses this information, and it’s immutably baked into the commits you start creating:
 
 ```
 git config --global user.name "<your-name>"
-git config --global user.email <your-email>
-
+git config --global user.email "<your-email>"
```
[the remark this answers is not in the document]
Comment (h3ndrk) [reply docs/setup/git_setup.md:12]: You mean the quotes or the empty line?

File: docs/framework/process_entrypoint.md (hunk 1 above)
Comment (h3ndrk) [reply docs/framework/process_entrypoint.md:28]: Iirc the behavior simulator is not a real HULK binary anymore. It works very differently currently. Therefore it would look weird if it would be included here.

File: docs/setup/git_setup.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (schluis) [reply docs/setup/git_setup.md:12]: No, I mean the """ ;)

**Review by schluis — APPROVED** [review]
